botcity/core/bot.py

```diff
@@ -35,6 +43,7 @@ class DesktopBot(BaseBot):
 
     def __init__(self):
         super().__init__()
+        self._app = None
```

Comment:
I just realized now that there is no getter/setter for the app.
Would you mind to add a `property` with the `setter` for app?
This will allow us to switch between apps in case where we need to manipulate multiple Desktop applications.

Reply:
Good point, I will add this